src/Pages/ProductDetail/RoomInformation/RoomInformation.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import styled from "styled-components";
+import InformationHeader from "./InformationHeader/InformationHeader";
+
+export default function RoomInformation({ roomInfo }) {
+  const { headerData } = roomInfo;
+  return (
+    <RoomInformationContainer>
+      <InformationContainer>
+        <InformationHeader headerData={headerData} />
```

Comment:
```suggestion
export default function RoomInformation({ roomInfo }) {
  return (
    <RoomInformationContainer>
      <InformationContainer>
        <InformationHeader headerData={roomInfo.headerData} />
```
취향 차이일 수는 있겠으나 저는 이 경우 이게 더 나은거 같습니다

Reply:
현재에는 {headerData}만 받아오는데 추후에 RoomCondition, description, bedType, Amenities 등 데이터 추가적으로 더 받아와서 해당하는 컴포넌트별로 Props로 내려주는 식으로 구성하고자 미리 써놨습니다 
최종형태는 아마도 {headerData, RoomCondition, description, bedType, Amenities} = roomInfo; 식으로 될 것 같습니다.